winpr/libwinpr/thread/thread.c

```diff
@@ -331,7 +328,6 @@ static void* thread_launcher(void* arg)
 			cleanup_handle(thread);
 	}
 	pthread_exit((void*) (size_t) res);
```

Comment:
Shouldn't we just return res here? Not returning anything might cause compiler warnings and simply returning res should result in an implicit pthread_exit(res); call according to docs.